.github/workflows/cr.yaml

```diff
@@ -12,11 +12,15 @@ jobs:
     steps:
       - name: Checkout Repo
         uses: actions/checkout@v3
-
+      - name: Run a multi-line script
+        run: |
+          echo "#######################################"
+          cat ${GITHUB_EVENT_PATH}
```

Comment:
It's generally not a good practice to print the entire `GITHUB_EVENT_PATH` as it might contain sensitive information. Consider parsing and printing only the necessary information.